decisions/websre/0002-web-sre-service-documentation.md

```diff
@@ -0,0 +1,48 @@
+Scope: Web SRE Team
+
+## Status
+
+Proposed
+
+## Context
+
+The Web SRE Team manages a number of services that vary in users, technologies used, setup, our commitment, infrastructure, process, etc. 
+
+Pre-existing documentation for these services, if it exists, also varies - in location, level of detail, scope, structure, presumed audience, etc.
+
+To help share knowledge between the Web SRE Team about these services' unique situations, a consistent baseline for our services documentation is a required first step.
+
+## Decision
+
+A Web SRE service here is defined as any codebase where we have some ownership over an aspect of this codebase being functional - e.g. the deployment process, automation, infrastructure running the service, observability of some aspect(s), or authoring the code ourselves. Web SRE services could be externally or internally-facing, including infrastructure services only Web SRE are aware of and leverage.
+
+For each Web SRE Service, we will author a Service Documentation page in Mana as a child of https://mana.mozilla.org/wiki/display/SRE/Service+Documentation.
+
+Service Documentation pages:
+* use the Web SRE Services Docs confluence page template for its structure;
+* have a primary audience of the Web SRE Team itself, with a secondary audience of other SRE teams;
+* are open to Mozilla internally, however not all teams of Mozilla are the intended audience of these pages;
+* have Runbook pages as child pages to the relevant Service Documentation page;
+* are maintained to reflect the current state - to the best of our knowledge - of a service's context, including a service being decommissioned (the Service Documentation page should note that but be left up as a tombstone marker of the decision);
+* cover Web SRE Services as defined above.
+
+Service Documentation pages do not:
+* include all possible services Web SRE might work with - e.g. we don't document AWS Services generically or services owned and maintained by other teams; 
+* include all possible details for all audiences beyond SRE - when such documentation requests come up, they can be added as a How To as a child to a service page, or a generic How To if not limited to one service;
+* live anywhere other than Mana. Web SRE Service documentation managed elsewhere, e.g. GitHub, Google Docs, should be migrated & then deprecated via links pointing readers to the Mana page;
+* replace other forms of documentation living elsewhere - e.g. codebase-specific documentation within a git repository, decision records in this repository, collaboration / draft developer notes in Google documents, Mana pages in other places and formats walking through shared infrastructure or processes, etc.;
+* entirely replace SRE_INFO.md files in Web SRE team-managed source code GitHub repositories. At a mininum, these SRE_INFO.md documents should be updated to point to the relevant Mana Page, and should be discussed and refined in a future ADR on those specifically.
```

Comment:
I searched for these via https://github.com/search?q=org%3Amozilla-it+filename%3Asre_info.md&type=code. It looks like 9 of them are for services we run. Checking the contents of a couple of them, the information they record is a subset of what we have in our mana documentation template. 

Given that, I would prefer to remove them and just link to mana instead.

Reply:
@sciurus feel free to do the removal work, make an add on adr about it, or add a proposed new line here.

Reply:
okay, made a change to the ADR to instead say service docs *should* replace SRE_INFO.md files. When encountered. the SRE_INFO.md should be reviewed for anything that needs to migrate to the service docs page, then be removed with a link to the service docs page added to the codebase readme.md. Is that sufficient, @sciurus ?